app/src/main/java/org/jellyfin/mobile/settings/ExternalPlayerType.java

```diff
@@ -0,0 +1,19 @@
+package org.jellyfin.mobile.settings;
+
+
+import androidx.annotation.StringDef;
+
+@StringDef({
+        ExternalPlayerType.MPV_PLAYER,
+        ExternalPlayerType.MX_PLAYER_FREE,
+        ExternalPlayerType.MX_PLAYER_PRO,
+        ExternalPlayerType.VLC_PLAYER,
+        ExternalPlayerType.SYSTEM_DEFAULT
+})
+public @interface ExternalPlayerType {
+    String MPV_PLAYER = "is.xyz.mpv";
+    String MX_PLAYER_FREE = "com.mxtech.videoplayer.ad";
+    String MX_PLAYER_PRO = "com.mxtech.videoplayer.pro";
+    String VLC_PLAYER = "org.videolan.vlc";
+    String SYSTEM_DEFAULT = "";
```

Comment:
```suggestion
    String SYSTEM_DEFAULT = "~system~";
```

*Note: '~' makes this an invalid Java package name, so you can use that.*